src/ripple/rpc/impl/Handler.cpp

```diff
@@ -146,6 +146,10 @@ Handler const handlerArray[]{
      byRef(&doSubmitMultiSigned),
      Role::USER,
      NEEDS_CURRENT_LEDGER},
+    {"server_definitions",
+     byRef(&doServerDefinitions),
+     Role::USER,
```

Comment:
Is `Role::User` the lowest privileged role in the system? I assumed `Role::GUEST` would be the lowest privilege amongst all of the `Roles`.

If that is the case, shouldn't `server_definitions` be invocable by `Role::GUEST` ?

Reply:
I don't see `Role::GUEST` used anywhere. `server_info` uses `Role::USER`, so I think that's the correct option here.